pkg/restapi/oidc/operations.go

```diff
@@ -507,64 +521,70 @@ func (o *Operation) userLogoutHandler(w http.ResponseWriter, r *http.Request) {
 	logger.Debugf("finished handling logout request")
 }
 
-func (o *Operation) onboardUser(sub, accessToken string) error { // nolint:funlen,gocyclo // not much logic
+func (o *Operation) onboardUser(sub, accessToken string) (string, error) { // nolint:funlen,gocyclo // not much logic
 	b := make([]byte, 32)
 
 	_, err := rand.Read(b)
 	if err != nil {
-		return fmt.Errorf("create user secret key : %w", err)
+		return "", fmt.Errorf("create user secret key : %w", err)
 	}
 
 	secrets, err := o.secretSplitter.Split(b, 2, 2)
 	if err != nil {
-		return fmt.Errorf("split user secret key : %w", err)
+		return "", fmt.Errorf("split user secret key : %w", err)
 	}
 
-	// TODO https://github.com/trustbloc/edge-agent/issues/488 send half secret key to hub-kms and remove logger
-	logger.Infof(string(secrets[1]))
+	walletSecretShare := base64.StdEncoding.EncodeToString(secrets[0])
+	hubAuthSecretShare := secrets[1]
```

Comment:
@Baha does it matter in what order we are combining secrets? Currently Key Server expects to get secret A in the header and fetch secret B from hub-auth. Here is the opposite situation. We probably should agree on order.

Reply:
@aholovko As far as I understand, the order doesn't matter and I was able to create keys locally (for now I have disabled the hub-auth integration as I am facing other issues).